### PR TITLE
Add Notice

### PR DESCRIPTION
**Pull request by petedejoy**
This PR disables the components on the CreateForm if user is in trial and already has a deployment. it also adds a notice as to why those components are disabled at the top of the form. I had to add an extra network request in here- there was no way around it, as the meaningful data (workspace) was being grabbed in the `Create` component before running the `createDeployment` mutation. I also changed the name of the `CreateForm` component- something was janky there.

### CODE REVIEW

File: src/modules/deployments/Deployment/Module.js [hunk 1]
```diff
@@ -32,8 +32,6 @@ const Metrics = Load(() =>
 
 const isLogs = str => str && str.toLowerCase() === 'logs'
```
Comment (petedejoy) [inline src/modules/deployments/Deployment/Module.js]: Just a formatting error my linter caught while exploring around.

File: src/modules/deployments/DeploymentConfigure/CreateForm.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 import { CardForm, Form, TextField, TextArea, FormSection } from 'instruments'
 
 import DeploymentConfig from '../Data/Config'
-
 import Executor from './Executor'
 import Info from './Info'
 import Usage from './Usage'
 import info from '../info'
 
-class Configure extends React.Component {
```
Comment (petedejoy) [inline src/modules/deployments/DeploymentConfigure/CreateForm.js]: I have _no_ idea why this was called `Configure` before, as it's imported to the `New` component as `CreateForm` 🤔

File: src/modules/deployments/New/index.js [hunk 3]
```diff
@@ -46,6 +52,8 @@ class New extends React.Component {
 
 New.propTypes = {
   location: PropTypes.object,
+  workspaces: PropTypes.array,
+  deployments: PropTypes.array,
 }
 
-export default New
+export default Data(New)
```
Comment (petedejoy) [inline src/modules/deployments/New/index.js]: Pass array of deployments down so we can get the workspace fragment that is returned with each deployment object (`billingEnabled`, `stripeCustomerId`)

File: src/modules/deployments/DeploymentConfigure/CreateForm.js (hunk 2 above)
Comment (schnie) [reply src/modules/deployments/DeploymentConfigure/CreateForm.js]: prob a copy/paste thing back in the day

File: src/modules/deployments/New/index.js [hunk 4]
```diff
@@ -34,10 +35,15 @@ class New extends React.Component {
       text: '',
       to: this.props.location.pathname,
     }
+    console.log(this.props.deployments)
```
Comment (schnie) [inline src/modules/deployments/New/index.js]: :eyes:

Comment (petedejoy) [reply src/modules/deployments/New/index.js]: NOOOO IVE BEEN SO GOOD

Comment (schnie) [reply src/modules/deployments/New/index.js]: The streak is broken!

**Comment by petedejoy** [comment]
@schnie don't merge this yet- there's a bug I gotta fix.

**Comment by petedejoy** [comment]
Ok- should be good to go.

**Pull request merged**